web-ui/src/main/resources/catalog/locales/du-admin.json

```diff
@@ -506,6 +506,12 @@
     "metadata.select": "selectie",
     "metadataAndTemplates": "Metadata & templates",
     "metadataFormatter": "formatter",
+    "metadataIdentifierTemplateName": "Name",
```

Comment:
You only need to update the english version. Transifex will take care of other languages on next update.